guide/onboarding/funding-a-wallet.md

```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/onboarding/funding-a-wallet/funding-request.png"
+   retina = "/assets/images/guide/onboarding/funding-a-wallet/funding-request@2x.png"
    width = 250
-   height = 550
-   alt-text = "Showing recovery phrase one word at a time"
-   caption = "Buy bitcoin screen from the [Wallet UI Kit](https://www.figma.com/file/VB3GQdAnhl8yta44DY3PSV/Bitcoin-Wallet-UI-Kit?node-id=1228%3A28550)"
+   height = 541
+   caption = "Show the user a funding on-chain payment request to fund their wallet."
+   alt-text = "Screen showing a funding payment request"
    layout = "float-left-desktop -background -shadow"
 %}
 
-As noted, most exchanges offer the option to buy bitcoin using fiat currencies. Once purchased, users can transfer those funds to a separate, external wallet. This might confuse new users who don't understand the difference between exchange wallets and personal wallets.
+Present the payment request to the user with an easily scannable QR code alongside having the option to copy and share it.
 
-Bitcoin ATMs are another great option for new users to exchange cash for bitcoin in person. These machines also let users trade their bitcoin for cash, which can be useful if they are somewhere new or foreign.
+Alongside the payment request, explain to the user that a payment channel will be opened with the received amount and that a minimum amount should be sent. This amount is to cover the network fees for opening a new payment channel.
 
-Finally, new users can buy bitcoin directly from other bitcoin users such as friends, other users at a meetup, or intermediary services like [localbitcoins](https://localbitcoins.com/).
+For users who want to [manually open a channel]({{'/guide/onboarding/funding-a-wallet/#manual-channel-opening' | relative_url}}) this screen should be skippable.
 
 </div>
 
-{% include /tip/open.html label="Tip: Teach users how to transfer" icon="info" color="blue" %}
+{% include /tip/open.html label="⚡ Lightning service" icon="info" color="blue" %}
 
-Users funding new wallets from an exchange could potentially come into contact with unknown concepts, such as a bitcoin address, QR code, or transaction confirmations. This is an excellent opportunity to explain the exchange to personal wallet flow.
+With Lightning services, the user can fund their wallet directly with a Lightning payment saving time and costs.
 
 {% include /tip/close.html %}
 
-{% include fact/pros.html %}
-  - An easy and popular way to acquire bitcoin
-{% include fact/close.html %}
+### Network sync
 
+If your application uses a method like compact block filters (BIP157/158) to sync to the Bitcoin network, which may take some time to sync, communicate to the user that their funds will not appear until the sync is complete. You may even consider not presenting a payment request until this sync is complete to prevent user confusion that their funds have not shown up.
```

Comment:
```suggestion
Sync methods like compact block filters (BIP157/158) to sync to the Bitcoin network take some time to finish. If your application uses them, you should communicate to the user that their funds will not appear until the sync is complete. You may even consider not presenting a payment request until this sync is complete. This can help prevent users from getting confused about their funds not showing up.
```
Tried to shorten sentences.